tests/Team.py

```diff
@@ -105,6 +105,20 @@ def testMembers(self):
         self.assertRaises(AssertionError, self.team.add_membership, user, "admin")
         self.team.remove_membership(user)
 
+    def testTeamMembership(self):
+        user = self.g.get_user("jacquev6")
+        self.assertListKeyEqual(self.team.get_members(), None, [])
```

Comment:
This feels awkward, just:

``self.assertEqual(list(self.team.get_members()), [])``

Reply:
done.